src/_playground/App.scss

```diff
@@ -1,243 +1,303 @@
-$mediaMaxWidth: 1280px;
+$breakpoint-lg: 1280px;
```

Comment:
Sorry for the massive diff in here.  I flattened the class structure which is why there are so many diffs.